components/classroom/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" type="text/css" href="./styles.css" />
+    <title>"TPEN Classroom Interface: Home"</title>
+</head>
+    <body>
+        <div class="header">
+            <h1>TPEN Classroom Interface</h1>
+            <div class="hamburger-menu" onclick="toggleMenu()">
+                &#9776 <!-- Hamburger icon -->
+            </div>
+        </div>
+            <div class ="navbar" id="navbar">
+            <!--navigation bar below: we can try making this collapsible on mobile devices?
+            placeholder categories, will be changed as we solidify the pages needed
+            -->
+            <ul>
+                <li><a href="#home">Home</a></li>
+                <li><a href="#annotations">Annotations</a></li>
+                <li><a href="#groups">Groups</a></li>
+                <li><a href="#contact">Contact</a></li>
+            </ul>
+            </div>
+
+            <div class="page-container">
+                <h1>Welcome!</h1>
+                <div class="input-container">
+                    <h2>Sign Up/In</h2>
```

Comment:
do we want to have two separate forms for signing up and logging in? if this works that's fine, just wondering bc signup might need more information from the user e.g. email address